searchobject/src/main/java/org/jalau/at18/searchobject/model/emotionrecognizer/EmotionRecognizer.java

```diff
@@ -11,28 +18,39 @@
 import java.util.Scanner;
 
 import org.apache.commons.io.FileUtils;
+import org.jalau.at18.searchobject.common.exception.EmotionRecognizerException;
+
 import com.google.gson.Gson;
+/**
+ *
+ *
+ * @throws EmotionRecognizerException if there is an error reading the file or with the http request
+ */
 
 public class EmotionRecognizer {
     private static final int JOY_POSITION = 12;
     private static final int SORROW_POSITION = 11;
     private static final int ANGER_POSITION = 10;
     private static final int SURPRISE_POSITION = 9;
     private  String[] result;
-    public EmotionRecognizer(String path, String token) throws IOException {
+    public EmotionRecognizer(String path, String token) throws EmotionRecognizerException {
         String imageBase64 = convertImage(path);
         InputStream inputStream = httpRequest(token, imageBase64);
         result = processStream(inputStream);
 
     }
 
-    private String convertImage(String filePath) throws IOException {
-        byte[] fileContent = FileUtils.readFileToByteArray(new File(filePath));
-        String encodedString = Base64.getEncoder().encodeToString(fileContent);
-        String firstPart = "{\n\"requests\":[\n{\n\"image\":{\n\"content\":\"";
-        String secondPart = "\"\n},\n\"features\":[\n{\n\"maxResults\":10,\n\"type\":\"FACE_DETECTION\"\n}\n]\n}\n]\n}";
-        String imageBase64 = firstPart + encodedString + secondPart;
+    private String convertImage(String filePath) throws EmotionRecognizerException {
+        try {
+            byte[] fileContent = FileUtils.readFileToByteArray(new File(filePath));
+            String encodedString = Base64.getEncoder().encodeToString(fileContent);
+            String firstPart = "{\n\"requests\":[\n{\n\"image\":{\n\"content\":\"";
+            String secondPart = "\"\n},\n\"features\":[\n{\n\"maxResults\":10,\n\"type\":\"FACE_DETECTION\"\n}\n]\n}\n]\n}";
+            String imageBase64 = firstPart + encodedString + secondPart;
         return imageBase64;
```

Comment:
Line 49-50 can be put together as `return firstPart + encodedString + secondPart;`